airbyte-metrics/metrics-lib/src/main/java/io/airbyte/metrics/lib/MetricAttribute.java

```diff
@@ -0,0 +1,10 @@
+/*
+ * Copyright (c) 2022 Airbyte, Inc., all rights reserved.
+ */
+
+package io.airbyte.metrics.lib;
+
+/**
+ * Custom tuple that represents a key/value pair to be included with a metric.
```

Comment:
thinking out loud, is it worth noting here (or anywhere else) how the different classes consume this?

Reply:
@davinchia My gut says to document the override in each client, as any comment we put here will quickly become out of date.  However, I will add that it is up to the client to determine what fields to use from this tuple.